backend/danswer/llm/chat_llm.py

```diff
@@ -228,6 +238,30 @@ def __init__(
     def log_model_configs(self) -> None:
         logger.debug(f"Config: {self.config}")
 
+    def _calculate_max_tokens(self, prompt: LanguageModelInput) -> int:
```

Comment:
nit: renaming to `_calculate_max_output_tokens`. `max_tokens` is a bit ambiguous